UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/ModelRunner.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using Barracuda;
+using UnityEngine.Profiling;
+
+namespace MLAgents.InferenceBrain
+{
+    public class ModelRunner : IBatchedDecisionMaker
+    {
+        private ITensorAllocator m_TensorAllocator;
+        private TensorGenerator m_TensorGenerator;
+        private TensorApplier m_TensorApplier;
+
+        private NNModel m_Model;
+        private InferenceDevice m_InferenceDevice;
+        private Model m_BarracudaModel;
+        private IWorker m_Engine;
+        private bool m_Verbose = false;
+        private string[] m_OutputNames;
+        private IReadOnlyList<TensorProxy> m_InferenceInputs;
+        private IReadOnlyList<TensorProxy> m_InferenceOutputs;
+
+        /// <summary>
+        /// Initializes the Brain with the Model that it will use when selecting actions for
+        /// the agents
+        /// </summary>
+        /// <param name="model"> The Barracuda model to load
+        /// <param name="brainParameters"> The parameters of the Brain used to generate the 
+        /// placeholder tensors
+        /// <param name="inferenceDevice"> Inference execution device. CPU is the fastest 
+        /// option for most of ML Agents models.
+        /// <param name="seed"> The seed that will be used to initialize the RandomNormal
+        /// and Multinomial objects used when running inference.</param>
+        /// <exception cref="UnityAgentsException">Throws an error when the model is null
+        /// </exception>
+        public ModelRunner(
+            NNModel model,
+            BrainParameters brainParameters,
+            InferenceDevice inferenceDevice = InferenceDevice.CPU,
+            int seed = 0)
+        {
+            m_Model = model;
+            m_InferenceDevice = inferenceDevice;
+            m_TensorAllocator = new TensorCachingAllocator();
+            if (model != null)
+            {
+#if BARRACUDA_VERBOSE
+                _verbose = true;
```

Comment:
I think this needs to be `m_verbose`. I bet the style enforcement renamed the original member but missed this inside the `#if`